Lighthouse2/Controllers/LoginController.cs

```diff
@@ -0,0 +1,13 @@
+using LBPUnion.Lighthouse2.Attributes;
+using Microsoft.AspNetCore.Mvc;
+
+namespace LBPUnion.Lighthouse2.Controllers;
+
+[ApiController]
+[GameRoute("login")]
+[Produces("text/plain")]
+public class LoginController : ControllerBase
```

Comment:
Only a suggestion, but should we have a separate GameController class like [the one I implemented in a PR](https://github.com/LBPUnion/ProjectLighthouse/blob/0e0a900f67136f2c9f0575ddc9d6ec5f729d3d78/ProjectLighthouse.Servers.GameServer/Types/GameController.cs) to reduce the duplication of the attributes, especially as we add more in the future.